third_party/move/move-compiler-v2/src/lib.rs

```diff
@@ -45,9 +45,9 @@ pub fn run_move_compiler(
             .and_then(|f| {
                 Path::new(f)
                     .file_name()
-                    .map(|f| f.to_string_lossy().to_string())
+                    .map(|f| f.to_string_lossy().as_ref().to_owned())
```

Comment:
Note that you don't need .as_ref() here.

Reply:
> Note that you don't need .as_ref() here.

No I do. That is because to_string_lossy returns not `&str` but some cow string.

Reply:
Cow<T> implements Fn to_owned(&self) -> T directly.

https://doc.rust-lang.org/std/borrow/enum.Cow.html

Reply:
Yeah but the result is not a string it is a `Cow<str>` ...

Reply:
BTW, if that `as_ref` would be unnecessary, cargo clippy would tell.

Reply:
Oop, you're right.  Sorry.